_docs/setup/create-app-usb.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Create App USBs
+title: Create app USBs
```

Comment:
The A in App USB is capitalized throughout the doc, either as "App USB" or "APP USB" (inconsistently). Shouldn't it be capitalized here?

Reply:
Corrected in a new commit.